cubi_isa_templates/isatab-stem_cell_core_sc/{{cookiecutter.__output_dir}}/s_{{cookiecutter.s_file_name}}.txt

```diff
@@ -1,8 +1,8 @@
-{% set sample_names = cookiecutter.sample_names.split(",") -%}
+{% set source_names = cookiecutter.source_names.split(",") -%}
```

Comment:
This actually is the sample name and *not* the source name

Reply:
It's the source_name variable being re-used to define sample names. Before it was the sample_name variable being re-used to define source names.

Reply:
You're right. Reverted this change.